README.md

```diff
@@ -50,15 +50,17 @@ artifact.platform_version # => "10.10"
 ```
 
 ## Unstable channel
-The `:unstable` channel is currently only available when connected to Chef's internal network.
-Configure Artifactory access by setting the following environment variables:
+The `:unstable` channel is currently only available when connected to Chef's internal network. mixlib-install uses a default set of read-only credentials when using :unstable channel. You can override these credentials by setting these environment variables:
+
 ```
 export ARTIFACTORY_USERNAME='username'
 export ARTIFACTORY_PASSWORD='password'
 ```
```

Comment:
I think we should keep this

On Fri, Apr 1, 2016 at 12:06 PM, Seth Chisamore <notifications@github.com>
wrote:

> In README.md
> <https://github.com/chef/mixlib-install/pull/90#discussion_r58252767>:
>
> > @@ -50,15 +50,17 @@ artifact.platform_version # => "10.10"
> >  ```
> >
> >  ## Unstable channel
> > -The `:unstable` channel is currently only available when connected to Chef's internal network.
> > -Configure Artifactory access by setting the following environment variables:
> > +The `:unstable` channel is currently only available when connected to Chef's internal network. mixlib-install uses a default set of read-only credentials when using :unstable channel. You can override these credentials by setting these environment variables:
> > +
> >  ```
> >  export ARTIFACTORY_USERNAME='username'
> >  export ARTIFACTORY_PASSWORD='password'
> >  ```
>
> Do we even need to offer this option? I think we should purge these env
> variables from the code base.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/mixlib-install/pull/90/files/c5ca59f741ef25bd0216a9a2972c5a16be241c0c#r58252767>
>